src/components/v2/ValueUpdate/styles.ts

```diff
@@ -0,0 +1,19 @@
+import { css } from '@emotion/react';
+import { useTheme } from '@mui/material';
+
+export const useStyles = ({ increase }: { increase: boolean }) => {
+  const theme = useTheme();
+
+  return {
+    container: css`
+      display: flex;
+      align-items: center;
+    `,
+    icon: css`
+      fill: ${increase ? theme.palette.interactive.success : theme.palette.interactive.error};
```

Comment:
Following my comment on using `color` instead of `fill` to be consistent:
```suggestion
      color: ${increase ? theme.palette.interactive.success : theme.palette.interactive.error};
```

Reply:
And are a few are styled with fill if this is the standard we want to hold